tests/upgrade/test_existence.py

```diff
@@ -0,0 +1,139 @@
+"""Upgrade TestSuite for validating Satellite entities existence and
```

Comment:
Yes ! So the plan is to move out everything in separate project called Satellite-Upgrade. This is just a POC to confirm that my approach is correct.